services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/services/resource_tracker_service_runs.py

```diff
@@ -125,3 +130,44 @@ async def list_service_runs(
         )
 
     return ServiceRunPage(service_runs_api_model, total_service_runs)
+
+
+async def export_service_runs(
+    s3_client: SimcoreS3API,
+    bucket_name: str,
+    user_id: UserID,
+    product_name: ProductName,
+    resource_tracker_repo: ResourceTrackerRepository,
+    wallet_id: WalletID | None = None,
+    access_all_wallet_usage: bool = False,
+    order_by: OrderBy | None = None,  # noqa: ARG001
+    filters: ServiceResourceUsagesFilters | None = None,  # noqa: ARG001
+) -> AnyUrl:
+    started_from = None
+    started_until = None
+    if filters:
+        started_from = filters.started_at.from_
+        started_until = filters.started_at.until
+
+    # Create S3 key name
+    s3_bucket_name = S3BucketName(bucket_name)
+    file_name = f"{datetime.now(tz=timezone.utc).date()}__{uuid4()}.csv"
```

Comment:
TIP: add a small prefix that identifies the document (ex. `sr_` as of *services runs*). Use a couple of letters similar to stripe identifiers. Also, i would add a short instead of a long uuid. this way you can use the same folder for other type of documents ...

Reply:
Ok, I can, but what identifies the document is already the path in the s3_object_key below: 
`s3_object_key = f"resource-usage-tracker-service-runs/{file_name}"` Therefore it is reusable for different use cases

Reply:
But I agree for convenient and debugging purposes it is a good practice! i will add it, thanks.